EthOn.rdf

```diff
@@ -305,11 +305,10 @@
 
     <owl:ObjectProperty rdf:about="http://ethon.consensys.net/hasBeneficiary">
         <rdfs:subPropertyOf rdf:resource="http://ethon.consensys.net/BlockObjectProperty"/>
-        <rdf:type rdf:resource="http://www.w3.org/2002/07/owl#FunctionalProperty"/>
         <rdfs:domain rdf:resource="http://ethon.consensys.net/Block"/>
         <rdfs:range rdf:resource="http://ethon.consensys.net/Account"/>
         <suggestedStringRepresentation>hasBeneficiary</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Relates a Block to the Account to which all fees collected from the successful mining of this Block are transferred.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Relates a Block to an Account to which fees or mining rewards from the successful mining of this Block are transferred.</rdfs:comment>
```

Comment:
Would it be more correct to say fees AND mining rewards? 
It could create the confusion of not being clear fees XOR mining rewards